### PR TITLE
Add project setup instructions

### PR DESCRIPTION
**Pull request by whitep4nth3r**
Closes https://github.com/whitep4nth3r/pantherworld/issues/13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,75 +1,71 @@
...
+## The game doesn't stop when the stream ends
 
-```bash
-# npm
-npm install
-
-# pnpm
-pnpm install
+p4nth3rworld is a multiplayer text-based game powered by a live coding stream at
+[twitch.tv/whitep4nth3r](twitch.tv/whitep4nth3r).
```
Comment (danielphilipjohnson) [inline README.md:8]: The link takes me too _https://github.com/whitep4nth3r/pantherworld/blob/2ca5648a622bf4f41740ef9a1e7e0a7ea00bb1e6/twitch.tv/whitep4nth3r_

Im guessing its due to there being no https:// 

[twitch.tv/whitep4nth3r](https://twitch.tv/whitep4nth3r)

**Comment by yoshivb** [comment]
Looks good to me albeit that this doesn't actually add any contribution guidelines in the form of a CONTRIBUTING.md.

**Comment by whitep4nth3r** [comment]
> Looks good to me albeit that this doesn't actually add any contribution guidelines in the form of a CONTRIBUTING.md.

Yeah I think that can come over time... I'd prefer *NOT* to write the guidelines and have someone else do it anyway — good first issue type of thing!

**Review by danielphilipjohnson — APPROVED** [review]

**Review by yoshivb — COMMENTED** [review]
👍

**Pull request merged**